src/poliastro/threebody/cr3bp_char_quant.py

```diff
@@ -0,0 +1,236 @@
+            print(
+                "KeyError-> Error in combination of bodies P1-P2, typo/DNE/system not created"
+            )
+            return 0, 0, 0, p1, p2
+        else:
+            tstar = self.__tstar_calc(mu_pi, lstar)
+            return mu, lstar, tstar, p1, p2
+
+    def __bodies_char(self):
+        """Returns mass ratio and mean distance of various P1-P2 systems
+
+        Returns
+        -------
+        mu_body: dict, float, km^3*s^-2
+        distances: dict, float, km
+        """
+        # Body values, G*M_body
+        mu_body = {}  # km^3 s^-2
+        mu_body["sun"] = general.GM_sun.to(u.km**3 * u.s**-2)
+        mu_body["mercury"] = general.GM_mercury.to(u.km**3 * u.s**-2)
+        mu_body["venus"] = general.GM_venus.to(u.km**3 * u.s**-2)
+        mu_body["earth"] = general.GM_earth.to(u.km**3 * u.s**-2)
+        mu_body["mars"] = general.GM_mars.to(u.km**3 * u.s**-2)
+        mu_body["jupiter"] = general.GM_jupiter.to(u.km**3 * u.s**-2)
+        mu_body["saturn"] = general.GM_saturn.to(u.km**3 * u.s**-2)
+        mu_body["uranus"] = general.GM_uranus.to(u.km**3 * u.s**-2)
+        mu_body["neptune"] = general.GM_neptune.to(u.km**3 * u.s**-2)
+        mu_body["pluto"] = general.GM_pluto.to(u.km**3 * u.s**-2)
+
+        mu_body["moon"] = general.GM_moon.to(u.km**3 * u.s**-2)
+        mu_body["phobos"] = general.GM_phobos.to(u.km**3 * u.s**-2)
+        mu_body["deimos"] = general.GM_deimos.to(u.km**3 * u.s**-2)
+        mu_body["europa"] = general.GM_europa.to(u.km**3 * u.s**-2)
+        mu_body["ganymede"] = general.GM_ganymede.to(u.km**3 * u.s**-2)
+        mu_body["enceladus"] = general.GM_enceladus.to(u.km**3 * u.s**-2)
+        mu_body["titania"] = general.GM_titan.to(u.km**3 * u.s**-2)
+        mu_body["triton"] = general.GM_triton.to(u.km**3 * u.s**-2)
+        mu_body["charon"] = general.GM_charon.to(u.km**3 * u.s**-2)
+
+        distances = {}  # km, diistance between the two primaries
+        distances["sunmercury"] = dist.mean_a_mercury
+        distances["sunvenus"] = dist.mean_a_venus
+        distances["sunearth"] = dist.mean_a_earth
+        distances["sunmars"] = dist.mean_a_mars
+        distances["sunjupiter"] = dist.mean_a_jupiter
+        distances["sunsaturn"] = dist.mean_a_saturn
+        distances["sunuranus"] = dist.mean_a_uranus
+        distances["sunneptune"] = dist.mean_a_neptune
+
+        distances["earthmoon"] = dist.mean_a_moon
+        distances["marsphobos"] = dist.mean_a_phobos
+        distances["marsdeimos"] = dist.mean_a_deimos
+        distances["jupitereuropa"] = dist.mean_a_europa
+        distances["jupiterganymede"] = dist.mean_a_ganymede
+        distances["saturnenceladus"] = dist.mean_a_enceladus
+        distances["saturntitan"] = dist.mean_a_titan
+        distances["uranustitania"] = dist.mean_a_titania
+        distances["neptunetriton"] = dist.mean_a_triton
+        distances["plutocharon"] = dist.mean_a_charon
+
+        return mu_body, distances
```

Comment:
This whole method is needed to translate body names to variable names. What about

- Having the primaries be `poliastro.bodies.Body` objects?
- Add `mean_a` as a property to the `Body` objects? (I know I have opposed to this in the past, but it's just convenience at this point)

Reply:
I can add the primaries be bodies.Body objects and add mean_a as a property. But for the moons like charon, titan, is it alright if I only add their: parent, k, mean_a?

Reply:
Yes 👍🏽